.github/CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ There are a number of guidelines considered when reviewing Pull Requests to be m
 
 <!-- Code -->
 
-[discord server]: https://sapphirejs.com/discord
+[discord server]: https://www.sapphirejs.dev/discord
```

Comment:
ok but we're NOT using www in our show off links

Reply:
updated